README.md

```diff
@@ -299,12 +312,20 @@ $ oauth2l fetch --scope cloud-platform,pubsub
 ### --audience
 
 The single audience to include in the signed JWT token. Required for jwt
-authentication type.
+authentication type. Can also be used for UAT.
 
 ```bash
 $ oauth2l fetch --type jwt --audience https://pubsub.googleapis.com/
 ```
 
+### --user_project
```

Comment:
quota_project to be consistent with gcloud

Reply:
Done